Include/internal/pycore_pyerrors.h

```diff
@@ -196,8 +196,16 @@ extern int _PyUnicodeError_GetParams(
     Py_ssize_t *objlen,
     Py_ssize_t *start,
     Py_ssize_t *end,
+    Py_ssize_t *len,
     int as_bytes);
 
+/* Specialization of _PyUnicodeError_GetParams(self, NULL, NULL, ...). */
+extern int _PyUnicodeError_GetSliceParams(
+    PyObject *self,
+    Py_ssize_t *start,
+    Py_ssize_t *end,
+    Py_ssize_t *len,
+    int as_bytes);
```

Comment:
AFAICS, this is only used in one file, in functions that are *themselves* specializations. It looks like it doesn't need a name.

Reply:
No you're right. I thought we would maybe need it later but I don't think so (I'll always need the underlying object). Removing it.